doc/source/development/dispatching.rst

```diff
@@ -16,6 +16,48 @@ and how the dispatching mechanism in scikit-image works.
 
     Expect the dispatching API to change without notice.
 
+Using backends
+--------------
```

Comment:
I don't think this is the right place for this section. It's aimed at backend users, while this document is aimed at backend developers. A good place to put this might be our user guide in `doc/source/user_guide`. What do you think?